capstone/src/main/webapp/script.js

```diff
@@ -12,6 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * The current possible tokenizers and their regular expressions to break up the text input. 
+ * @enum {regexp}
+ */
+const TokenizerEnum = {
+  WORD: /\S+\s*/g, 
+  SENTENCE: /([^\.!\?]+[\.!\?]+)|([^\.!\?]+$)/g,
```

Comment:
We might also want to include newlines as sentence breaks.

Reply:
Fixed